### PR TITLE
add(Install, HW and SW Reqts): add instructions to patch Chrome 60

### PR DESCRIPTION
**Pull request by ugaston**
Versions: Proposal for 7.5 (should be adapted for all versions starting from 6.2)

Set of instructions to patch Bonita BPM Platform for Chrome 60+ versions, both for Subscription and Community users.

_Reference bug: **BS-16954**- Portal incompatible with Chrome 60 (released 25 Jul 2017)

### CODE REVIEW

**Comment by tbouffard** [comment]
@ugaston can you please sign the CLA?

**Comment by tbouffard** [comment]
@ugaston does this modification should also be applied to doc 7.3 and 7.4? I have just seen you created the pull request on the 7.5 branch.

**Comment by ugaston** [comment]
Basically, the procedure to patch the portal has to be documented for all versions starting from 6.2.

This is (version-specific details notwithstanding) the procedure to communicate, but the Hardware and Software Requirements page has evolved slightly over the versions, so the actual placement of the procedure may need adapting for the previous versions.

File: md/hardware-and-software-requirements.md [hunk 1]
```diff
@@ -50,5 +50,74 @@ Notes:
...
+1. Zip all the files and folders again into 'portal-theme.zip'. 
+   (BEWARE: make sure not to zip the 'portal-theme' folder, but its contents. If the 'portal-theme.zip' contains a 'portal-theme' folder at the base, the Bonita Portal will not recognise it as a valid zip structure.)
+1. In the portal, click on 'Import and apply a new Look&Feel' and choose the updated 'portal-theme.zip' file.
+1. Back in your Chrome update 60 browser, empty the cache.
+
+#### Community users
+
+1. In an installed Bonita bundle, edit file 'server/webapps/bonita/portal/scripts/includes/common.js':
+   1. Add the code below after or before the existing code:
+   ```java
```
Comment (abirembaut) [inline md/hardware-and-software-requirements.md]: should be javascript

**Review by abirembaut — CHANGES_REQUESTED** [review]
LGTM
Just one small comment

**Comment by ugaston** [comment]
Thank you for your comment, Anthony. Good catch.
I have now replaced "java" with "javascript", as suggested.

File: md/hardware-and-software-requirements.md [hunk 2]
```diff
@@ -50,5 +50,74 @@ Notes:
...
+:::
+
+1. Log in the Bonita Portal as Administrator.
+1. In the menu, click on 'Portal'.
+1. Click on 'Export the current Look&Feel'.
+1. Make a back-up copy of the exported file.
+1. Unzip the exported file 'portal-theme.zip'.
+1. Edit 'BonitaConsole.html' file:
+   1. In the `<head>` section, add the code below:
+   ```xml
```
Comment (laurentleseigneur) [inline md/hardware-and-software-requirements.md]: this script block is `javascript`, not `xml`

File: md/hardware-and-software-requirements.md [hunk 3]
```diff
@@ -50,5 +50,74 @@ Notes:
...
+   <script>
+      // Monkey Patch xhr
+      // Due to a specification change in the xhr.getAllResponseHeaders method Bonita Portal does not behave as expected 
+      // in browsers that implement this new specification (currently only Chrome >60).
+      // This patch fixes xhr.getAllResponseHeaders unwanted behavior whithin Bonita Portal context
```
Comment (sperfect) [inline md/hardware-and-software-requirements.md]: within

**Comment by ugaston** [comment]
Thank you, Laurent and sperfect (?) for your comments.
My last commit should have taken good care of those.

**Review by abirembaut — APPROVED** [review]

**Comment by abirembaut** [comment]
LGTM

**Pull request merged**